src/runtime_src/core/pcie/driver/linux/xocl/subdev/icap.c

```diff
@@ -2134,7 +2142,7 @@ static int icap_verify_bitstream_axlf(struct platform_device *pdev,
 
 done:
 	if (err)
-		xocl_dyn_subdevs_destory(xdev);
+		xocl_subdev_destroy_by_level(xdev, XOCL_SUBDEV_LEVEL_URP);
```

Comment:
I think we can remove this call. At this moment, the xclbin is already downloaded to hardware. We can keep the subdevs been created successfully. They will be removed when download next xclbin, switch PLP, etc.

Reply:
done